docs/faq/frequently_asked_questions.rst

```diff
@@ -0,0 +1,56 @@
+
+#. I’m trying to use DroneCAN but my motor isn’t spinning.
+    * A common issue that may cause your module not to spin is that its parameters are not configured correctly. Please confirm that you have properly configured the :ref:`Motor Direction <throttle_direction>`, :ref:`Mode <throttle_mode>`, and :ref:`Maximums <throttle_maximums>` for your module. Also, confirm that an :ref:`ESC Index <dronecan_px4_tutorial_esc_index>` is set for DroneCAN. If your module is not using :ref:`Arming Bypass <dronecan_arming_and_bypass>`, confirm that you are sending DroneCAN commands that will arm the module initially. By default, commands close to 0% throttle should arm the module. For more information on how to use DroneCAN with your module, please refer to our :ref:`DroneCan Integration with PX4 and Ardupilot <dronecan_fc_tutorial>` documentation.
+
+#. What hardware do I need prior to receiving my module?
+    * Certain modules come with included hardware, such as prop adapters and heatshrink. Please refer to your module’s family page found in the sidebar under the Modules section for everything that is included in the box.
+    * You will need screws to attach a propeller to your module and to attach your module to the drone arm. Please check your module’s datasheet found on the modules family page under “Documentation” on our `website <https://www.vertiq.co/>`_. It is also highly recommended to apply a threadlocker, such as Loctite 243, to your screws when attaching a propeller to your module.
+    * For all modules, you will need a UART-to-Serial adapter in order to communicate with your module using your computer. We recommend using a `CP2102 FTDI adapter <https://www.amazon.com/Ximimark-Module-Serial-Converter-CP2102/dp/B07T1XR9FT?>`_.
+
+#. Why is my module slowing down, even though I am not changing the throttle or increasing the throttle? Why is my module derating?
+    * Each motor has built in safeties to protect the module from overheating failures. The module will start to spin slower, or derate, when the safeties are in effect to keep it at a safe operating speed. To learn more about these safety features, please refer to the :ref:`Safety Systems <manual_safety>` documentation.
+    * Each module has a max-continuous thrust region depending on the size of the propeller you are using. Please refer to your module’s `Thrust Data <https://www.vertiq.co/thrust-data>`_ to learn more about its max-contintinous thrust.
```

Comment:
why would this cause me to slow down?

Reply:
Do you mean: "Why would derating cause me to slow down?" ?

Reply:
No, sorry. I mean why would the module having a maximum continuous thrust region cause me to slow down? What even is a maximum continuous thrust region?

Reply:
Maximum continuous torque and thrust regions are the setpoints you can operate in without overheating. Go outside of them, and you overheat and start to derate. That may not be obvious to customers, so maybe good to just define why those regions are relevant with respect to heat.